src/Polly.RateLimiting/README.md

```diff
@@ -10,13 +10,11 @@ Example:
 
 ``` csharp
 // Convenience extension method for ConcurrencyLimiter
-builder.AddConcurrencyLimiter(
-    new ConcurrencyLimiterOptions
-    {
-        PermitLimit = 10,
-        QueueLimit = 10
-    },
-    () => Console.WriteLine("Rate limiter rejected!"));
+builder.AddConcurrencyLimiter(new ConcurrencyLimiterOptions
```

Comment:
Let's add a sample for the most basic concurrency limiter extension with one/two `int` parameters

Reply:
Done.